deepchem/models/optimizers.py

```diff
@@ -678,3 +677,44 @@ def _create_jax_schedule(self):
     return optax.piecewise_constant_schedule(
         init_value=self.initial_rate,
         boundaries_and_scales=self.boundaries_and_scales)
+
+
+class KFAC(Optimizer):
+  """The Second order gradient optimiation algorithm which uses an approximation to calculate the inverse of the Fischer matrrix"""
+
+  def __init__(self, **kwargs):
+    """
+    Parameters:
+    -----------
+    model: torch.nn.Module
+    The model to be optimized.
```

Comment:
The description line for each argument needs to be indented.  See https://numpydoc.readthedocs.io/en/latest/format.html#docstring-standard.